pebble_manager.py

```diff
@@ -40,6 +42,9 @@ def request_running_app(self):
         # This is an appmessage with a null UUID and dictionary {2: 1} with a uint8 value.
         self.pebble._send_message("LAUNCHER", "\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x02\x00\x00\x00\x02\x01\x00\x01")
 
+    def is_timeline_supported(self):
```

Comment:
I wonder if this could just be a property (you can make them by using the `@property` decorator on a function). If so, I'd rename it `timeline_is_supported`. If left as a method, the current name is fine.